website/source/docs/auth/aws-ec2.html.md

```diff
@@ -1005,8 +1005,16 @@ in its identity document to match the one specified by this parameter.
 <dl class="api">
   <dt>Description</dt>
   <dd>
-    Creates a role tag on the role. Role tags provide an effective way to restrict the
-    capabilities that are set on the role.
+Creates a role tag on the role, which help in restricting the capabilities that
+are set on the role. Role tags are not tied to any specific ec2 instance unless
+specified explicitly using `instance_id` parameter. By default, role tags are
+designed to be used across all instances that satisfies the constraints on the
+role. Regardless of which instances have role tags on them, the restriction
```

Comment:
I'd reword this as "Regardless of which instances have role tags on them, capabilities defined in a role tag must be a strict subset of the given role's capabilities."